encoding.go

```diff
@@ -3,16 +3,39 @@ package encoding
 import (
 	"github.com/signalfx/golib/trace"
 	"time"
+	"strconv"
+	"fmt"
 )
 
+// low, high
+type ID [2]uint64
+
+func GetID(s string) ID {
+	var low, high uint64
+	if len(s) == 16 {
+		low, _ = strconv.ParseUint(s, 16, 64)
+	} else if len(s) == 32 {
+		high, _ = strconv.ParseUint(s[:16], 16, 64)
```

Comment:
is the input string system generated, if not then do we want to check for err during parsing?

Reply:
Fair. I'll update.